### PR TITLE
[BACK-68] Get number of contributors for a given repository

### PR DESCRIPTION
**Pull request by acreturs**
### Description of Issue
Function is designed to get all the contributors for a given repository including the amount of contributions that person has already done for this repository.

### How I implemented
I use a simple Rest api call to retreive the information. Everything is stored within an array that contains the name of the contributor and the amount of contributions done by that person.

### Other (any extra notes, additional features, problems?,

### CODE REVIEW

**Review by kigadye — APPROVED** [review]
Improve comments, otherwise good

**Review by thomas-woehrle — CHANGES_REQUESTED** [review]
good job on the test file.

But the function does not do what it is supposed to. As the description of the Linear task says it should return the number of contributors. Instead it seems like it returns the top 30 contributors. Your request probably only fetches the first 30 entries of the contributors. 

So rewrite the function to return the number of contributors. You should also put the function into the file githubApi.ts because scraping != calling an API.

The link I mentioned in the linear task also shows a way to only use one api call instead of counting all the individual contributors. Maybe this works still if not find a different way.

**Review by thomas-woehrle — CHANGES_REQUESTED** [review]
Small things. I also did a commit to get rid of the console.log statements and to replace the old getContributorCount method with yours. 
So pull before continue working. 
Also add authorization!

File: packages/graphql_server/src/api/githubApi.ts [hunk 1]
```diff
@@ -262,3 +203,31 @@ export async function getRepositoryTopics(
...
+
+export async function getContributorCount(owner: string, repo: string): Promise<number> {
+  try {
+    const response = await axios.get(`https://api.github.com/repos/${owner}/${repo}/contributors`, {
+      params: {
+        per_page: 1
+      }
+    })
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```
[the remark this answers is not in the document]
Comment (acreturs) [reply packages/graphql_server/src/api/githubApi.ts:223]: For some reason I still get any. I will just receive it as a string.

[the remark this answers is not in the document]
Comment (acreturs) [reply packages/graphql_server/src/api/githubApi.ts:223]: const linkHeader: string = response?.headers?.link as string

File: packages/graphql_server/src/api/githubApi.ts [hunk 2]
```diff
@@ -262,3 +203,31 @@ export async function getRepositoryTopics(
...
+ * returns the amount of contribtuors. Not 100% accurate since the gitHub API does not return certain contributors and on the other hand
+ * sometimes shows new contribtuors that can not be seen on the github page.
+ * @param owner
+ * @param repo
+ * @returns array of strings containing the name of the contributor and the number of commits done by that perso
+ */
+
+export async function getContributorCount(owner: string, repo: string): Promise<number> {
+  try {
+    const response = await axios.get(`https://api.github.com/repos/${owner}/${repo}/contributors`, {
```
[the remark this answers is not in the document]
Comment (acreturs) [reply packages/graphql_server/src/api/githubApi.ts:217]: you mean for the response?

File: packages/graphql_server/src/api/githubApi.ts [hunk 3]
```diff
@@ -262,3 +203,31 @@ export async function getRepositoryTopics(
...
+      params: {
+        per_page: 1
```
[the remark this answers is not in the document]
Comment (acreturs) [reply packages/graphql_server/src/api/githubApi.ts]: I am not sure how to add anon as a parameter

**Review by thomas-woehrle — APPROVED** [review]
Nice job Alex!

**Pull request merged**